eclair-core/src/main/resources/reference.conf

```diff
@@ -13,6 +13,7 @@ eclair {
     binding-ip = "127.0.0.1"
     port = 8080
     password = "" // password for basic auth, must be non empty if json-rpc api is enabled
+    use-new-version = false
```

Comment:
I would name this `use-old-api` and set to false (make the api default).

Reply:
Done via 8268e70